app/institutions/dashboard/-components/object-list/template.hbs

```diff
@@ -0,0 +1,57 @@
+<IndexCardSearcher
+    @queryOptions={{@defaultQueryOptions}}
+    @sort={{this.sort}}
+    @page={{this.page}}
+as |list|>
+    <Institutions::Dashboard::-Components::InstitutionalDashboardWrapper @institution={{@institution}} as |wrapper|>
+        <wrapper.main local-class='main-column'>
+            {{#if list.searchObjectsTask.isRunning}}
+                <LoadingIndicator @dark={{true}} />
+            {{else}}
+                <table data-test-object-list-table local-class='object-table'>
+                    <thead local-class='object-table-head'>
+                        <tr>
+                            {{#each @columns as |column|}}
+                                <th local-class='object-table-header'>
+                                    {{column.name}}
+                                </th>
+                            {{/each}}
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        {{#each list.searchResults as |result|}}
+                            <tr>
+                                {{#each @columns as |column|}}
+                                    <td>
+                                        {{#if (eq column.type 'link')}}
+                                            <Institutions::Dashboard::-Components::ObjectList::LinkField
+                                                @searchResult={{result}}
+                                                @getHref={{column.getHref}}
+                                                @getLinkText={{column.getLinkText}}
+                                            />
```

Comment:
i'm not necessarily opposed to the `::LinkField` component, but with the `{{call}}` helper you could invoke `<OsfLink>` in-place, as you had before

Reply:
Yea, I think I will see about using `{{call}}` for it. No need for extra components if they're not needed